bot.py

```diff
@@ -283,59 +283,61 @@ def check(bot, remind=False):
         reminder = "Reminder: " if remind else ""
 
         info = EAT_REACTIONS[choice]
-        message = (
-            f"<!everyone> {reminder}We're eating {info['desc']}! " f"{info['instr']}"
-        )
+        message = f"<!everyone> {reminder}We're eating {info['desc']}! {info['instr']}"
 
-        if info["type"] == DeliveryType.bike:
+        if info["type"] == DeliveryType.BIKE:
             message += f"\n{lowest} has the honour to :bike: today"
-        elif info["type"] == DeliveryType.delivery:
+        elif info["type"] == DeliveryType.DELIVERY:
             message += f"\n{lowest} has the honour to pay for this :money_with_wings:"
 
         bot.chat_post_message(channel, message)
 
     except KeyError as e:
         bot.chat_post_message(
             channel,
-            "Oh no something went wrong. "
-            "Back to the manual method, @pingiun handle this!",
+            "Oh no something went wrong. Back to the manual method, @pingiun handle this!",
         )
         raise e
 
 
-def usage():
-    """Prints usage"""
-    print(f"Usage: {sys.argv[0]} [ post | check | remind ]", file=sys.stderr)
-    sys.exit(1)
-
-
-def main():
-    global WBW_EMAIL
-    global WBW_PASSWORD
-    global WBW_LIST
+if __name__ == "__main__":
+    argument_parser = argparse.ArgumentParser("What Are We Eating Today Slack Bot")
 
-    if len(sys.argv) != 2:
-        usage()
+    argument_parser.add_argument(
+        "--wbw-email", help="The username of the WieBetaaltWat user.", required=True
+    )
+    argument_parser.add_argument(
+        "--wbw-password", help="The password of the WieBetaaltWat user.", required=True
+    )
+    argument_parser.add_argument(
+        "--wbw-list", help="The WieBetaaltWat list.", required=True
+    )
 
-    WBW_EMAIL = os.environ["DJANGO_WBW_EMAIL"]
-    WBW_PASSWORD = os.environ["DJANGO_WBW_PASSWORD"]
-    WBW_LIST = os.getenv("WBW_LIST", "e52ec42b-3d9a-4a2e-8c40-93c3a2ec85b0")
+    argument_parser.add_argument(
+        "--slack-token", help="The Slack API Token.", required=True
+    )
+    argument_parser.add_argument(
```

Comment:
The reason that the secrets were supplied as env vars is that supplying secrets on the command line is not safe

Reply:
It's best practice to supply secrets as environment variables or in files because command line arguments can be viewed by all users using `ps`

Reply:
This program runs as a cronjob with the passwords still passed as arguments.